### DISCUSSION TITLE
自定义参数问题

### DISCUSSION THREAD

**Discussion by Y827748053** [w7corp/easywechat#2420] [General]
请问下群发接口的原创文章校验参数send_ignore_reprint默认是多少，能手动更改配置么

**Comment by overtrue** [comment]
> 当 send_ignore_reprint 参数设置为1时，文章被判定为转载时，且原创文允许转载时，将继续进行群发操作。
> 当 send_ignore_reprint 参数设置为0时，文章被判定为转载时，将停止群发操作。
> send_ignore_reprint 默认为0。
https://developers.weixin.qq.com/doc/offiaccount/Message_Management/Batch_Sends_and_Originality_Checks.html

麻烦先看官方文档后再使用。
